docs/topics/OBST-vertx-jdbc-rest-introduction.adoc

```diff
@@ -0,0 +1,29 @@
+[[vertx-crud-introduction]]
+= Introduction
+
+This Quickstart showcases simple database interaction in a Cloud environment.
+The example application exposes an HTTP API, which provides “CRUD” endpoints to manipulate data through Create, Read, Update, and Delete operations.
```

Comment:
Also, 

Create, Read, Update, and Delete operations.

to 

_create_, _read_, _update_, and _delete_ operations.